idb_path: "Dropbox/aims_db/sim_space_lab.idb"
odb_path: "Dropbox/aims_db/sim_space_lab.odb"


gravity: [0, 0, -9.79343]

time_calibrate: 0
time_start: 7.4
time_end: 90.
dt: 0.0001

features:
  min_init_confidence: 0.25
  max_missed_frames: 10
  new_keyframe_threshold: 5

# camera parameters
camera:
  fx: 432. # pixels
  fy: 432. # pixels
  cx: 317. # pixels
  cy: 252. # pixels
  pixel_noise: 0.6
  distortion: [-0.29932978711199204, 0.09469642895565503, 8.98894755601866e-05, 0.000298086422736973, 0.0]


# imu parameters
imu:
  gyro_noise: 0.001
  gyro_walk: 0.00004
  accel_noise: 0.015
  accel_walk: 0.0002

rangefinder:
  range_noise: 0.003

# intersensor parameters
intersensor:
  p_c_i: [0.0508, -0.0508, 0.]
  p_r_i: [0.145, 0.025 , 0.044]
  q_ic: [0.5, 0.5,  0.5,  0.5]
  q_ir: [0, 0, 0, 1.]

# initial conditions
x0:
  p_i_w: [1.807055819651424, -1.6072152417759167, 0.1375364509287777]
  v_i_w: [0.,  0.,  0.] # m/s
  q_wi: [-0.6557497755536573, 0.7548727650763649, -0.004917501384813389, 0.01124362980262305]


  gyro_bias: [0.,0.,0.] # rad/s
  accel_bias: [0.,0.,0.] # m/s^2


# initial 3 sigma bounds
x0_3_sigma:
  p_i_w: 0.001 # m
  v_i_w: 0.0001 # m/s
  theta_wi: 0.001 # degrees
  gyro_bias: 0.001 # rad/s
  accel_bias: 0.001 # m/s^2
